modules/storages/app/models/storages/share_point_storage.rb

```diff
@@ -30,9 +30,11 @@
 
 module Storages
   class SharePointStorage < Storage
-    store_attribute :provider_fields, :tenant_id, :string
-
-    def self.visible? = false
+    # For now SharePoint is visible only in tests.
+    # This is to prevent it from being shown in the UI, as it is not ready yet.
+    def self.visible?
+      Rails.env.test?
```

Comment:
Nice touch. =)

Reply:
Moving to `Rails.env.local?` in the next PR 😄